src/relayers/relayersDataController.ts

```diff
@@ -21,7 +21,9 @@ export class RelayersController extends Controller {
     }
 
     @Get("")
```

Comment:
Can you please add doc comments that describe this as being as a unix timestamp?